manifest/process.go

```diff
@@ -37,6 +37,16 @@ func NewProcess(app string, s Service, m Manifest) Process {
 		}
 	}
 
+	for i, p := range s.Ports {
+		// Assign random ports to "internal" balancers
+		if !p.Public {
+			p.Public = true
```

Comment:
@mattmanning with a private balancer we no longer bind to the host at all so this doesn't actually break.

This does mean you can't access these services from the host but that should be fine?

Could you test master against a repo you know has this issue?

I would like to check this isn't an issue